cms/models.py

```diff
@@ -792,6 +814,18 @@ def get_context(self, request, *args, **kwargs):
             else None,
             "enrolled": enrolled,
             "user": request.user,
+            # The context variables below are added to avoid duplicate queries within the templates
+            "background_video_url": self.background_video_url,
+            "testimonials": self.testimonials,
+            "faculty": self.faculty,
+            "course_lineup": self.course_lineup,
+            "course_pages": self.course_pages,
+            "for_teams": self.for_teams,
+            "faqs": self.faqs,
+            "outcomes": self.outcomes,
+            "who_should_enroll": self.who_should_enroll,
+            "techniques": self.techniques,
+            "propel_career": self.propel_career,
```

Comment:
If we're going to have all of this repeated code between `CoursePage` and `ProgramPage`, we should have a class/mixin that they inherit from, or at the very least we should have a test that runs `get_context` for both classes and asserts that they have the same keys to prevent these two implementations from drifting apart

Reply:
Moved to `ProductPage` so this isn't duplicated.